src/python/test/test_dxpy.py

```diff
@@ -197,6 +198,7 @@ def test_update_describe(self):
         self.assertEqual(desc["restricted"], True)
         self.assertEqual(desc["downloadRestricted"], True)
         self.assertEqual(desc["description"], "new description")
+        self.assertEqual(desc["allowedExecutables"][0], "applet-FGkbg180f5v2p1226jFqGJGy")
```

Comment:
Is it an arbitrary ID? To emphasize it I'd write something like: "applet-123456..." 

AFAIR we check on the server side if the hash is correctly formatted but not if the executable exists? That is fine.

Minor: please add a test somewhere here for removing/nulling allowed executables list.